ArduCopter/mode_rtl.cpp

```diff
@@ -309,7 +309,7 @@ void ModeRTL::descent_run()
     attitude_control->input_thrust_vector_heading_cd(pos_control->get_thrust_vector(), auto_yaw.get_heading());
 
     // check if we've reached within 20cm of final altitude
-    _state_complete = labs(rtl_path.descent_target.alt - copter.inertial_nav.get_position_z_up_cm()) < 20;
+    _state_complete = labs(rtl_path.descent_target.alt - pos_control->get_pos_estimate_NEU_cm().tofloat().z) < 20;
```

Comment:
probably don't need tofloat() here?

Reply:
I will check if the compiler complains.

Reply:
There were a number of places that this was done. All fixed.